README.md

```diff
@@ -11,43 +11,25 @@
 
 The AKS Karpenter Provider enables node autoprovisioning using [Karpenter](https://karpenter.sh/) on your AKS cluster.
 
-## Status of Project:
-
 The API for AKS Karpenter Provider is currently alpha (`v1alpha2`).
 
-## Development
-
-A [GitHub Codespaces]((https://github.com/features/codespaces)) development flow is described below, which you can use to test karpenter functionality on your own cluster, and to aid rapid development of this project.
-
-1. **Install VSCode**: Go [here](https://code.visualstudio.com/download) to download VSCode for your platform. After installation, in your VSCode app install the "GitHub Codespaces" Extension. See [here](https://code.visualstudio.com/docs/remote/codespaces) for more information about this extension.
-
-2. **Create Codespace** (~2min): In browser, click Code / "Create a codespace on main" (for better experience customize to use 4cores/8GB), wait for codespace to be created. It is created with everything needed for development (Go, Azure CLI, kubectl, skaffold, useful plugins, etc.) Now you can open up the Codespace in VSCode: Click on Codespaces in the lower left corner in the browser status bar region, choose "Open in VSCode Desktop". (Pretty much everything except for `az login` and some `az role assignment` works in browser; but VSCode is a better experience anyway.)
-
-More information on GitHub Codespaces is [here](https://github.com/features/codespaces).
-
-3. **Provision cluster, build and deploy Karpenter** (~5min): Set `AZURE_SUBSCRIPTION_ID` to your subscription (and customize region in `Makefile-az.mk` if desired). Then at the VSCode command line run `make az-all`. This logs into Azure (follow the prompts), provisions AKS and ACR (using resource group `$CODESPACE_NAME`, so everything is unique / scoped to codespace), builds and deploys Karpenter, deploys sample `default` Provisioner and `inflate` Deployment workload.
```

Comment:
I have some thoughts on the readme, but mostly agree with removing all of these steps, but we need to have a way to reference how to create a cluster with node auto provisioning enabled, and how to create a self hosted karpenter cluster. 

These belong in both the website and the readme. 

For now, can we leave the readme as is, and I will add a followup PR for creating a readme with the style of kueue. https://github.com/kubernetes-sigs/kueue?tab=readme-ov-file#kueue

Reply:
https://github.com/Azure/karpenter/pull/71